### PR TITLE
Medium Articles Details Scrapper

### PR DESCRIPTION
**Pull request by kaustubhgupta**
Issue:  #92 
This script will generate a CSV file containing details about various medium articles published by a specific publication on a given date range. The number dates can be specified and all dates are chosen randomly. 

image

### CODE REVIEW

File: Web-Scraping/Medium-Articles-Details-Scrapping/run.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+from app import Scrap
+
+
+a = Scrap(urls_dict={"Towards Data Science": "https://towardsdatascience.com",
```
Comment (SKAUL05) [inline Web-Scraping/Medium-Articles-Details-Scrapping/run.py]: Use Dynamic Inputs instead of hardcoded ones

**Review by SKAUL05 — CHANGES_REQUESTED** [review]
Looks very good. CQ is very good. Follow the changes requested.

File: Web-Scraping/Medium-Articles-Details-Scrapping/run.py (hunk 1 above)
Comment (Neilblaze) [reply Web-Scraping/Medium-Articles-Details-Scrapping/run.py]: +1 from my side!

File: Web-Scraping/Medium-Articles-Details-Scrapping/app/__init__.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+import random
+import numpy as np
+import itertools
+import time
+
+class Scrap:
+    """
```
Comment (Neilblaze) [inline Web-Scraping/Medium-Articles-Details-Scrapping/app/__init__.py]: @kaustubhgupta it'll be recommended if you shift this comment part to the README as it's description part!

**Review by Neilblaze — CHANGES_REQUESTED** [review]

**Comment by kaustubhgupta** [comment]
Hey @SKAUL05 & @Neilblaze , I have made the required changes. Please review and let me know if it can be merged now.

image

**Comment by powerexploit** [comment]
> Hey @SKAUL05 & @Neilblaze , I have made the required changes. Please review and let me know if it can be merged now.
> 
> image

@kaustubhgupta Nice contributions also add the screenshot in the folder.
Then we I @ankitdobhal (PA) will merge the PR.
Thanks

**Comment by kaustubhgupta** [comment]
@ankitdobhal Done with the screenshot!

**Review by SKAUL05 — APPROVED** [review]

**Pull request merged**